python/lsst/sims/GalSimInterface/galSimInterpreter.py

```diff
@@ -821,6 +828,15 @@ class carrying all of the information for the object whose image
             # Poisson distribution.
             obj = centeredObj.withFlux(realized_flux)
 
+            use_fft = False
+            if realized_flux > 1.e6 and fft_sb_thesh is not None and realized_flux > thresh:
```

Comment:
A couple typos on this line I think:
`if realized_flux > 1.e6 and fft_sb_thresh is not None and realized_flux > fft_sb_thresh:`